packages/sdk/src/mint/mint.ts

```diff
@@ -5,11 +5,28 @@ import { NearContractCall } from '../execute';
 
 export type MintArgs =  {
   nftContractId?: string;
-  reference: string;
   ownerId: string;
+  metadata: TokenMetadata;
```

Comment:
This is a breaking API change, perhaps we can still accept reference, then make an additional property in `options` called `additionalMetadata` , then in args: `metadata: { reference, ...additionalMetadata }`

Reply:
Did we decide not to do any breaking changes? Shouldn't typing + versioning be enough?

Reply:
Fair point, when you're in alpha/beta anything server `0.x` technically all bets are off.  Maybe we release beta this week with the clear message "you should pin versions and look for changes to this API each time you update"

Reply:
I left it in flexibly now, but I prefer the `metadata.reference` API. Requires some logic to check if it's conflicting or not. If we prioritize API, let's remove it.

Reply:
The current version I pulled still had the breaking change, removed the conflict message and updated the docs.  I think we will just roll the breaking change as part of beta release task today